final-react/.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.env
```

Comment:
I think I added this in my other waiting PR? Needs to be in there though, thanks